phpstan.neon.dist

```diff
@@ -1,4 +1,4 @@
 parameters:
-    level: 7
+    level: 5
```

Comment:
any reason of this change? I'd suggest adding a phpstan baseline instead 
- https://phpstan.org/user-guide/baseline

Reply:
also I'd suggest to add php 8 in github actions workflow CI 

https://github.com/LinioIT/seller-center-sdk/blob/a3c6c60d3f6af3c6d1ca6a6dc91f7cbd77be8af5/.github/workflows/CI.yml#L14